lib/server/getUserByLoginToken.js

```diff
@@ -0,0 +1,40 @@
+const USER_TOKEN_PATH = 'services.resume.loginTokens.hashedToken';
+const NO_VALID_USER = undefined;
+
+let Accounts;
+
+try {
+  // `accounts-base` is a weak dependency, so we'll try to require it
+  // eslint-disable-next-line global-require, prefer-destructuring
+  Accounts = require('meteor/accounts-base').Accounts;
```

Comment:
Maybe move this require to inside `getUserByLoginToken` like you do with the require for `apollo-link-http` and `body-parser `. That way we can simply let the require call throw.